src/promptflow/promptflow/_sdk/_service/utils.py

```diff
@@ -20,8 +22,26 @@ def wrapper(*args, **kwargs):
             response.status_code = 404
             return response
         except Exception:  # pylint: disable=broad-except
-            response = jsonify({"error_message": "Internal Server Error"})
+            response = jsonify({"error_message": f"Internal Server Error"})
             response.status_code = 500
             return response
 
     return wrapper
+
+
+def validate_request_user():
```

Comment:
note we will need to infrom vsc extension on this. 
Maybe we only add such check for connection calls?

Reply:
Only connection api will validate the header.